### ISSUE TITLE
embObjMotionControl linking error

### ISSUE THREAD

**Issue by mebbaid** [robotology/icub-main#529]
After a clean build of `YARP` and `icub` to the latest `devel` we get this error when calling `yarpdev --list` on the head

```
error while opening /usr/local/src/robot/icub-main/build/lib/iCub/embObjMotionControl.so:
  /usr/local/src/robot/icub-main/build/lib/iCub/embObjMotionControl.so: undefined symbol: _ZN4yarp3dev25ImplementAmplifierControlINS0_19embObjMotionControlENS0_17IAmplifierControlEE9enableAmpEi
[WARNING]Wrong library name for plugin embObjMotionControl
```
@S-Dafarra @traversaro @marcoaccame

**Comment by traversaro** [comment]
cc @Nicogene @valegagge

**Comment by Nicogene** [comment]
Fixed by 1506c0357eb62708ee059a502aeefaec358f14fb
